dura/trust_path/template/theme.php

```diff
@@ -1,11 +1,12 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"> 
-<html xmlns="http://www.w3.org/1999/xhtml"> 
+<!DOCTYPE html>
+<html lang="en" xml:lang="en">
 <head>
-<meta http-equiv="content-type" content="text/html; charset=utf-8" /> 
+<meta charset="utf-8">
 <meta name="viewport" content="width = 620" />
 <title><?php e(t(DURA_TITLE)) ?> | <?php e(t(DURA_SUBTITLE)) ?></title>
-<link href="<?php echo DURA_URL; ?>/css/style.css" rel="stylesheet" type="text/css" media="screen" />
-<script type="text/javascript" src="http://www.google.com/jsapi"></script> 
+<link rel="stylesheet" href="<?php echo DURA_URL; ?>/css/style.css"/>
```

Comment:
Just a suggestion for consistency, either remove the slashes on meta, link, etc. or just add ` />` space slash greater than.